### DISCUSSION TITLE
Is it possible to install only some of the boost components?

### DISCUSSION THREAD

**Discussion by Smalldy** [microsoft/vcpkg#34551] [Q&A]
I just want to use boot-system boost-filesystem, but I found too much boost*.a file in vcpkg_installed dir.

**Comment by dg0yt** [comment]
I assume you installed `boost-filesystem` and `boost-system` only, not `boost`.
These ports have dependencies which need to be installed, too.
You can inspect these dependencies at the command line, e.g.
~~~
vcpkg depend-info boost-system
~~~
or online, e.g.
https://vcpkg.link/ports/boost-system
